deepchecks/core/serialization/suite_result/widget.py

```diff
@@ -151,7 +151,7 @@ def prepare_failures(
             children = (HTML(value='<p>No outputs to show.</p>'),)
         else:
             df = create_failures_dataframe(failures)
-            table = DataFrameSerializer(df.style.hide_index()).serialize()
+            table = DataFrameSerializer(df.style.hide(axis='index')).serialize()
```

Comment:
we require `pandas>=1.1.5` but I think `Styler.hide()` was added in the latest versions which means that this code might fail/

note: I fixed it in my PR in the next way 
```python
styler = styler.hide(axis='index') if hasattr(styler, 'hide') else styler.hide_index()
table = DataFrameSerializer(styler).serialize()
```

Reply:
from [docs](https://pandas.pydata.org/docs/reference/api/pandas.io.formats.style.Styler.hide.html)

> New in version 1.4.0.

Reply:
Understood, so I'm closing this